videos/migrations/0022_auto_20180910_1143.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('videos', '0021_tag_is_ap_tag'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='video',
+            name='category',
+            field=models.ForeignKey(default=-1, to='videos.Category'),
```

Comment:
I'm not sure this thing will work, since this will give the category_id default value of -1, the foreign key constraint might break since the Category table does not have any id = -1.